packages/valory/services/apy_estimation/service.yaml

```diff
@@ -22,7 +22,7 @@ models:
         spooky_subgraph:
         - '0x2a651563c9d3af67ae0388a5c8f89b867038089e'
         - '0x2b4c76d0dc16be1c31d4c1dc53bf9b45987fc75c'
-      history_start: ${SERVICE_APY_HISTORY_START:int:null}
+      n_observations: ${SERVICE_APY_N_OBSERVATIONS:int:120}
```

Comment:
Set a default of 120 observations which is a good number to start experimenting with. Moreover, now the operator does not need to specify anything if they do not need to update the number of observations. Previously, they had to update the `history_start` every time they restarted the service which was cumbersome.

Reply:
yes. There probably is some range that should be allowed, not up to infinity

Reply:
In aea-config.yaml n_observations is 10 while here it is 120. Is that right?

Reply:
Agreed. I plan to extend this method at some point, though we might want to introduce a proper way for validation to all of our models.
https://github.com/valory-xyz/open-autonomy/blob/554269bf20fe1f4ded081fa81833b1563d59b63c/packages/valory/skills/apy_estimation_abci/models.py#L209-L220

Have you heard of [pydantic](https://pydantic-docs.helpmanual.io/)? Using it might produce some cool results.